### PR TITLE
feat(author-ids) auto generate inventair links

### LINKED ISSUE #8518: Auto-generated Inventaire.io author links from Wikidata ids
Note: not totally sure this is a good idea (though it's growing on me) but it seemed like it might be easy to implement so I tried it so now we have a prototype for people to try.

Inventaire.io builds mostly on wikidata data, and gives it a slightly more book-focused look:

https://www.wikidata.org/wiki/Q42
https://inventaire.io/entity/wd:Q42

### Describe the problem that you'd like solved
We are able to auto-generate links to Inventaire.io for any author where we have a wikidata ID.

### Proposal & Constraints
On output of ids on the Author page, immediately after the wikidata id, add a link to Inventaire generated from the wikidata id.

Inventaire can also have non-wikidata ids, so we should check if an Inventaire id has been added manually, and skip the second output.

### Additional context
Screenshot 2023-11-11 at 10 25 43

### PR DESCRIPTION
**Pull request by davidscotson**
Closes #8518 

feature

### Testing
Visit an author with a wikidata ID, see the new link created belowe the wikidata link.

### Screenshot
Screenshot 2023-11-11 at 10 25 43

### CODE REVIEW

**Comment by tfmorris** [comment]
Since this is just a link decorator, it seems like something that would be more appropriate as a browser plugin which  Inventaire.io subscribers could use on any site with Wikidata links.

If OpenLibrary is going to start privileging certain sites with publicity via links, I think it would be useful to have a set of public criteria which are used to determine whether or not a site qualifies. Otherwise it seems totally random when Better World Books or Inventaire.io get included and other sites don't.

**Comment by cdrini** [comment]
@tfmorris I see these no different as adding identifiers. Identifiers let us link to other parts of the internet and make our data more useful by allowing folks to use us as a 'normalizing' layer between many disparate services. We're generally not super picky ; if the link provides value we tend to include it.

This specific case is interesting because we aren't storing the inventaire IDs in open library, but just linking to them.

@davidscotson Does inventaire have its own set of identifiers under the hood? Or does it just use Wikidata?

**Comment by davidscotson** [comment]
Inventaire has it's own identifiers for things that are not in Wikidata. I assume they 'upgrade' to wikidata over time when possible.

Not every book belongs in Wikidata since they have their own inclusion criteria:

> For an item to be notable by Wikidata's policies, it must meet at least one of three criteria: Having a presence in any Wikimedia project (Wikipedia, Wikimedia Commons, Wikiquote, etc.) Making reference to a concept or object that is clearly identifiable. Validating statements made in other existing items.

Inventaire appear to have a wikidata passthrough, you can search for almost anything and it shows up, with some mild intelligence e.g.:

https://inventaire.io/entity/wd:Q152088

detects that it's not a book or author and redirects to a subject page for "French Fries":

https://inventaire.io/entity/wdt:P921-wd:Q152088

But I think that they'll show something sensible for anything that has book related data (authors, works, editions, series) in wikidata, which I assume applies to almost everything that has a wikidata link in Open Library.

**Review by cdrini — APPROVED** [review]
Ok then I think this seems reasonable? The only awkward bit is that this won't appear in the `.json` version of the page, and won't be queriable from solr via `id_inventaire:*` like our other IDs. But I reckon that's ok; it's still useful to link back and have the `sameAs` acknowledgement!

File: openlibrary/templates/type/author/view.html [hunk 1]
```diff
@@ -188,6 +188,9 @@ <h3>$_('ID Numbers')</h3>
                         $if id.name in page.remote_ids:
                             $ href = id.url.replace('@@@', page.remote_ids[id.name])
                             <li>$id.label: <a itemprop="sameAs" href="$href">$page.remote_ids[id.name]</a></li>
+                            $if id.name == 'wikidata' and 'inventaire' not in page.remote_ids:
+                              $ inventaire_href = href.replace('wikidata.org/wiki/', 'inventaire.io/entity/wd:')
+                              <li>Inventaire.io: <a itemprop="sameAs" href="$inventaire_href">wd:$page.remote_ids[id.name]</a></li>
```
Comment (cdrini) [inline openlibrary/templates/type/author/view.html:193]:
```suggestion
                              <li>Inventaire.io: <a
                                  itemprop="sameAs"
                                  href="https://inventaire.io/entity/wd:$page.remote_ids[id.name]"
                              >wd:$page.remote_ids[id.name]</a></li>
```

**Pull request merged**